src/sentry/notifications/notifications/active_release.py

```diff
@@ -0,0 +1,242 @@
+                        },
+                        release=last_release,
+                    )
+            notify(provider, self, participants, shared_context)
+
+    def get_context(self) -> MutableMapping[str, Any]:
+        environment = self.event.get_tag("environment")
+        enhanced_privacy = self.organization.flags.enhanced_privacy
+        group = self.group
+        context = {
+            "project_label": self.project.get_full_name(),
+            "group": group,
+            "users_seen": self.group.count_users_seen(),
+            "event": self.event,
+            "link": get_group_settings_link(
+                self.group, environment, rule_details=None, referrer="alert_email_release"
+            ),
+            "has_integrations": has_integrations(self.organization, self.project),
+            "enhanced_privacy": enhanced_privacy,
+            "last_release": self.last_release,
+            "last_release_link": self.release_url(self.last_release),
+            "last_release_slack_link": self.slack_release_url(self.last_release),
+            "environment": environment,
+            "slack_link": get_integration_link(self.organization, "slack"),
+            "has_alert_integration": has_alert_integration(self.project),
+            "regression": self.event_state.is_regression
+            if self.event_state and self.event_state.is_regression
+            else False,
+        }
+
+        # if the organization has enabled enhanced privacy controls we don't send
+        # data which may show PII or source code
+        if not enhanced_privacy:
+            contexts = (
+                self.event.data["contexts"].items() if "contexts" in self.event.data else None
+            )
+            event_user = self.event.data["event_user"] if "event_user" in self.event.data else None
+            context.update(
+                {
+                    "tags": self.event.tags,
+                    "interfaces": get_interface_list(self.event),
+                    "contexts": contexts,
+                    "event_user": event_user,
+                }
+            )
+
+        return context
+
+    def record_active_release_notification_sent(
+        self,
+        participant: Team | User,
+        event: Event,
+        provider: ExternalProviders,
+        release_version: str,
+    ) -> None:
+        suspect_committer_ids = [
+            go.owner_id()
+            for go in GroupOwner.objects.filter(
+                group_id=self.group.id,
+                project=self.project.id,
+                organization_id=self.project.organization_id,
+                type=GroupOwnerType.SUSPECT_COMMIT.value,
+            )
+        ]
+        code_owner_ids = [o.id for o in get_owners(self.project, event)]
+        team_ids = (
+            [t.id for t in Team.objects.get_for_user(self.organization, participant)]
+            if type(participant) == User
+            else None
+        )
+
+        analytics.record(
```

Comment:
@barkbarkimashark Wondering why we need so much data for analytics and why we can't use the built-in analytics that the base notification class provides

Reply:
Ahh, didn't know we had that functionality in the base class .. updated.

Not sure if analytics events is the right place for it, but we wanna be able to query the events and see which participants might have been better versus the release committers.